src/ui/public/visualize/visualize_legend.html

```diff
@@ -33,21 +40,31 @@
             ng-show="canFilter(legendData)"
           >
             <button
-              class="kuiButton kuiButton--basic kuiButton--small" ng-click="filter(legendData, false)"
+              class="kuiButton kuiButton--basic kuiButton--small"
+              ng-click="filter(legendData, false)"
+              aria-label="Filter for value {{legendData.label}}"
             >
               <span class="kuiIcon fa-search-plus"></span>
             </button>
 
             <button
-              class="kuiButton kuiButton--basic kuiButton--small" ng-click="filter(legendData, true)"
+              class="kuiButton kuiButton--basic kuiButton--small"
+              ng-click="filter(legendData, true)"
+              aria-label="Filter out value {{legendData.label}}"
             >
               <span class="kuiIcon fa-search-minus"></span>
             </button>
           </div>
 
-          <div class="legend-value-color-picker">
+          <div class="legend-value-color-picker" role="listbox">
+            <span id="{{legendId}}ColorPickerDesc" class="kuiScreenReaderOnly">Set color for value {{legendData.label}}</span>
```

Comment:
Can we indent this?

```html
<span
  id="{{legendId}}ColorPickerDesc"
  class="kuiScreenReaderOnly"
>
  Set color for value {{legendData.label}}
</span>
```

Reply:
What if I would say "no"? Yes of course we can :-)

Reply:
Then you better have a good reason! :D